src/components/TagSet/TagSetFilter.tsx

```diff
@@ -89,7 +101,7 @@ export const TagSetFilter = chakra((props: TagSetFilterProps) => {
                   align="right"
                   name="close"
                   size="small"
-                  color="ui.gray.x-dark"
+                  color={dismissButtonColor}
```

Comment:
I'm surprised these work. There is a specific set of colors that can be used as defined in `IconColors`, but realistically, all these colors work. We should update `IconColors` type in a later PR.

Reply:
Yes, there have been a number of additions to the global color palette, but we have not update the available colors in `Icon`.  Here's a [ticket](https://jira.nypl.org/browse/DSD-1181).